uint8 robotID
rosMsgs/s_robot robotPos
---
